scenarios/kubernetes_bazel.py

```diff
@@ -240,6 +244,10 @@ def create_parser():
         '--gcs-shared',
         default="gs://kubernetes-jenkins/shared-results/",
         help='If $PULL_REFS is set push build location to this bucket')
+    parser.add_argument(
+        '--ci',
```

Comment:
bikeshed: call it something like `--publish-file`? `--ci` can be confusing :-\

Reply:
SGTM

Reply:
I did --publish-version